bacs/src/main/java/com/adyen/checkout/bacs/BacsDirectDebitComponentParams.kt

```diff
@@ -8,15 +8,17 @@
 
 package com.adyen.checkout.bacs
 
+import androidx.annotation.RestrictTo
 import com.adyen.checkout.components.base.ComponentParams
 import com.adyen.checkout.components.model.payments.Amount
 import com.adyen.checkout.core.api.Environment
 import kotlinx.parcelize.Parcelize
 import java.util.Locale
 
 @Parcelize
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 // TODO check if this class is still needed once all params support [Amount]
```

Comment:
So is this still needed?

Reply:
Yep, we didn't add the amount to all configs yet

Reply:
Whoops my bad, somehow I thought this was saying to check again after the config classes could be removed 😅